.eslintrc.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  "extends": [
+    "react-app",
+    "prettier/@typescript-eslint",
+    "plugin:prettier/recommended"
+  ],
+  "settings": {
+    "react": {
+      "version": "999.999.999"
```

Comment:
What's this?

Reply:
This is from the tsdx defaults:
https://github.com/formium/tsdx/blob/master/src/createEslintConfig.ts

Just a validation for major, minor, patch